tests/test_colour_filters.py

```diff
@@ -0,0 +1,19 @@
+import sys
+import os
+import pytest
+
+from pymagine import colour_filters as cf
+
+fname = os.path.join(os.path.dirname(__file__), '../tests/imgs/coronado_beach.jpeg')
+bad_ftype = 'imgs/coronado_beach.csv'
+url_fname = 'https://upload.wikimedia.org/wikipedia/commons/thumb/c/c3/Python-logo-notext.svg'
+
+def test_inputs():
```

Comment:
Remember to add docstrings for the test function
```suggestion
def test_inputs():
"""
Applies tests to the colour_filters function to ensure proper usage.
"""
```